flycheck.el

```diff
@@ -6079,7 +6079,18 @@ See URL `http://handlebarsjs.com/'."
           "Error: Parse error on line " line ":" (optional "\r") "\n"
           (zero-or-more not-newline) "\n" (zero-or-more not-newline) "\n"
           (message) line-end))
-  :modes (handlebars-mode handlebars-sgml-mode))
+  :modes (handlebars-mode handlebars-sgml-mode web-mode)
+  :predicate (lambda ()
+               (if (eq major-mode 'web-mode)
+                   (and (boundp 'web-mode-engine)
+                        (boundp 'web-mode-engine-file-regexps)
+                        (string-equal web-mode-engine "ctemplate")
```

Comment:
Both `boundp` checks are necessary to prevent compilation warning about free variables.